pubsub/tests/unit/pubsub_v1/publisher/batch/test_thread.py

```diff
@@ -201,6 +202,27 @@ def test_blocking__commit_wrong_messageid_length():
         assert isinstance(future.exception(), exceptions.PublishError)
 
 
+def test_block__commmit_api_error():
+    batch = create_batch()
+    futures = (
+        batch.publish({'data': b'blah blah blah'}),
+        batch.publish({'data': b'blah blah blah blah'}),
+    )
+
+    # Make the API throw an error when publishing.
+    error = google.api_core.exceptions.InternalServerError('uh oh')
+    types.PublishResponse(message_ids=['a'])
```

Comment:
Why are we creating a `PublishResopnse` instance here?

Reply:
copypasta. Fixed f53ccbff6.